pyproject.toml

```diff
@@ -32,10 +31,11 @@ classifiers = [
 "Changelog" = "https://github.com/seapagan/simple-toml-settings/blob/main/CHANGELOG.md"
 
 [tool.poetry.dependencies]
-python = ">=3.9,<4.0"
+python = ">=3.10,<4.0"
 rtoml = ">=0.9.0,<1.0.0"
 github-changelog-md = ">=0.9.0"
 
+xdg-base-dirs = "^6.0.1"
```

Comment:
I don't know why poetry is putting libs so they have to be manually adjusted here

Reply:
> I don't know why poetry is putting libs so they have to be manually adjusted here

yeah it tends to use the "^" spec which can be a pain, though dependabot adjusts this as needed when there are upgrades. I use a poetry plugin called 'poetry-relax' periodically which does as it sounds.